src/main/java/Calculator.java

```diff
@@ -0,0 +1,45 @@
+import java.util.ArrayList;
+
+public class Calculator {
+    private Formatter formatter = new Formatter();
+    private int howManyPeople;
+    private ArrayList<Product> products; // создал массив
```

Comment:
🍏 Ссылки на эти объекты (а в случае howManyPeople  - и значение) не будут меняться, поэтому можно объявить их final, чтобы защитить от случайных изменений:
```suggestion
    private final int howManyPeople;
    private final ArrayList<Product> products;
```